src/script/api/script_window.hpp

```diff
@@ -706,7 +706,7 @@ class ScriptWindow : public ScriptObject {
 		 * Extra viewport; %Window numbers:
 		 *   - Ascending value = #ExtraViewportWidgets
 		 */
-		WC_EXTRA_VIEW_PORT                           = ::WC_EXTRA_VIEW_PORT,
+		WC_EXTRA_VIEWPORT                            = ::WC_EXTRA_VIEWPORT,
```

Comment:
If these are now auto-generated, should they still be under version control?

I tried not editing the file (as suggested by the comment at the top of the `WindowClass` enum); however, the project failed to build.

Reply:
Ugh, never mind. I forgot to `git fetch` :facepalm: